components/deploy/DeploymentEvents.tsx

```
@@ -0,0 +1,21 @@
+interface DeploymentEventsProps {
+  logs: any[];
+}
+
+export function DeploymentEvents({ logs }: DeploymentEventsProps) {
+  if (!logs || !Array.isArray(logs) || logs.length === 0) return null;
+  return (
+    <div className="mt-4 text-left max-h-64 overflow-y-auto border-t pt-2">
+      <h4 className="text-sm font-semibold mb-2">Latest Build Logs</h4>
+      <ul className="text-xs font-mono space-y-1">
+        {logs.map((log: any, idx: number) => (
+          <li key={idx}>
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)